src/resources.rs

```diff
@@ -189,6 +194,16 @@ pub fn add_tcp_stream(stream: tokio::net::TcpStream) -> Resource {
   Resource { rid }
 }
 
+pub fn add_hyper_body(body: hyper::Body) -> Resource {
+  let rid = new_rid();
+  let mut tg = RESOURCE_TABLE.lock().unwrap();
+
+  let body = HttpBody::from(body);
+  let r = tg.insert(rid, Repr::HttpBody(body));
```

Comment:
Cool. Where does the body get removed from the table?

Reply:
Uh - that's a very good point and potential glaring oversight on my part... I need to investigate.

Reply:
Fixed - added a close method.